event/csv_handler.go

```diff
@@ -38,11 +42,13 @@ func (handler CSVHandler) Handle(event *DimensionsInserted) error {
 	log.Debug("getting file", log.Data{"url": url, "event": event})
 	readCloser, err := handler.fileGetter.Get(url)
 	if err != nil {
+		handler.errorHandler.Handle(event.InstanceID, err)
```

Comment:
There's a lot of stutter in this function call. Perhaps the field on CSVHandler should just be `errors` so this call becomes handler.errors.Handle?